src/sync.cpp

```diff
@@ -3285,7 +3286,7 @@ void JSONSyncConfigIOContext::serialize(const SyncConfig& config,
     writer.arg_B64("n", config.mName);
     writer.arg_B64("tp", config.mOrigninalPathOfRemoteRootNode);
     writer.arg_fsfp("fp", config.mLocalFingerprint);
-    writer.arg("th", config.mRemoteNode, MegaClient::NODEHANDLE);
+    writer.arg("th", config.mRemoteNode, sizeof(handle));
```

Comment:
Careful, a `handle` is 8 bytes, but nodehandles are 6 bytes (the 2 MSB are not used).
```suggestion
    writer.arg("th", config.mRemoteNode,  MegaClient::NODEHANDLE);
```

Reply:
I've just noticed that `deserialize()` also reads 8 bytes, so it should be fine. FYI, in the `sctable`, node handles are always stored as 6 bytes, but it makes sense in order to save some bytes. For syncs, I think it's not a concern.
Closing this discussion ;)